### PR TITLE
[IMP] Rename cost_price to 'Replenishment cost'

### LINKED ISSUE #6: `product_get_cost_field` and `stock` both use "Cost price" to name different fields
As a workaround for https://github.com/odoo/odoo/issues/3396, I think the module `product_get_cost_field` should change the label of `product.product.standard_price` to something else but "Cost price", to make it clear what each field does.

PS: as it turns out, the wording in standard module `stock` is probably right and `product_get_cost_field` should not add a field labeled "Standard Price".
To avoid breaking existing installations, in the short term the field `cost_price` should be relabeled to another name.
In the longer term, we should strive to integrate `product_get_cost_field`'s features deeper into the standard costing features.

### PR DESCRIPTION
**Pull request by clonedagain**
Fixes #6
This change is meant as a short-term solution, to avoid confusion between standard_price and cost_price since both are named 'Cost Price' in the GUI.
The accounting best practises seem to be that if a Cost price is known, it must be used for the valuation of the stock. So the right 'Cost price' field is standard_price and we have to name cost_price something else.
The description of the inherited fields is changed too, because for some reason the server seems unable to export the new names in the .po files.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I'm still with you :+1:

File: product_cost_incl_bom/__openerp__.py [hunk 1]
```diff
@@ -31,11 +31,11 @@
 Product Cost including BOM costs
```
Comment (yvaucher) [inline product_cost_incl_bom/__openerp__.py]: I think this title could go in `summary` and it could be replaced by Manufacturing cost

**Comment by clonedagain** [comment]
After more thinking (really I can never get enough thinking sorry), I think it should actually be called "Replenishment Cost", because that would let me cleanly introduce a module to optionally base the value on purchases instead of standard prices.
@pedrobaeza would that suit you?

**Comment by clonedagain** [comment]
@yvaucher I factored your comment in too.

**Comment by pedrobaeza** [comment]
OK, seems good. Please put a big help about the use of the field.

**Comment by clonedagain** [comment]
I've extended the help text, does it suit you now please?

**Comment by clonedagain** [comment]
I've pushed an early version of the extension I'd like to propose which requires the name change in #12.

**Comment by pedrobaeza** [comment]
It looks good to me, :+1:, waiting only that Travis doesn't say the contrary.

**Comment by clonedagain** [comment]
not sure whether the remaining travis errors are related or not...

**Comment by pedrobaeza** [comment]
Yeah, Travis problems are unrelated.

**Comment by guewen** [comment]
:+1: 

Thanks!

**Pull request merged**